types/network/network.go

```diff
@@ -21,6 +22,13 @@ type IPAMConfig struct {
 	AuxAddress map[string]string `json:"AuxiliaryAddresses,omitempty"`
 }
 
+// IPAMInfo represents IPAM operational information
+type IPAMInfo struct {
+	PoolID  string `json:",omitempty"`
```

Comment:
`PoolID` is an index the ipam driver provides to libnetwork in response to pool requests for future lookups when requesting/releasing addresses.
It is not an information for the docker end user/app. It is not meant for that, it is not part of the operational information. I don't think it should be included in this structure.